README.md

```diff
@@ -97,6 +97,8 @@ The build script will check for a `build.conf` file in the target image director
 
 * **version**: The version to use for tagging. Default: `1.0.0`. Note: that additionally, the CircleCI build number is always appended to the version as a revision (for example: `1.0.0.15519`) to create a unique version per build.
 * **devonly**: If set the image will be pushed only to the `devdemisto` org in docker hub and will not be pushed to the `demisto` org. Should be used for images which are for development purposes only (such as the image used in CircleCI to build this project).
+* **deprecated**: If set the image will be listed as deprecated in the deprecated_images.json file and the image will be forbidden form using in the integrations/automations.
+* **deprecated-reason**: Free text that explain the deprecation reason.
```

Comment:
Isn't this supposed to be `deprecated_reasons` ?

Reply:
yes you are right